### DISCUSSION TITLE
Accordion

### DISCUSSION THREAD

**Discussion by nevoben1** [vikramlearning/blazorbootstrap#551] [Q&A] [answered]
hello ,
i was wondering if there is any way to set the accordion title text location to the middle of the component ?
in addition , is there any way to set everything to be right to left instead of left to right ? 
thank you so much !

**Comment by gvreddy04** [comment, marked as answer]
> hello , i was wondering if there is any way to set the accordion title text location to the middle of the component ? 

Right now not supported. We need to check on this.

> in addition , is there any way to set everything to be right to left instead of left to right ? 

If you are referring to RTL support, please refer https://getbootstrap.com/docs/5.3/getting-started/rtl/.
